hack/jenkins/build_changelog.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Copyright 2022 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script can take the following env variables
+# ARGS: args to pass into the make rule
+# 	OUTPUT_LOCATION = the location to write the changelog file to
+
+set -x -o pipefail
+
+if (($# < 1)); then
+  echo "ERROR: given ! ($#) parameters but expected 1."
+  echo "USAGE: ./build_changelog.sh OUTPUT_LOCATION"
+  exit 1
+fi
+
+OUTPUT_LOCATION=${1}
+gh search prs --merged --sort updated --limit 100 --repo kubernetes/minikube --json number,title,closedAt --template '{{range .}}{{tablerow (printf "#%v" .number | autocolor "green") .title (timeago .closedAt)}}{{end}}' > ${OUTPUT_LOCATION}
```

Comment:
```suggestion
gh search prs --merged --sort updated --limit 100 --repo kubernetes/minikube --json number,title,closedAt --template '{{range .}}{{tablerow (printf "#%v" .number | autocolor "green") .title (timeago .closedAt)}}{{end}}' > "${OUTPUT_LOCATION}"
```

As per ShellCheck: `Double quote to prevent globbing and word splitting.`

Reply:
Done